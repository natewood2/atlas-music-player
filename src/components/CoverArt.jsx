import React from "react";
import Cover from "/images/cover-art.png";

export default function CoverArt({ src }) {
  return (
    <div className="w-full overflow-hidden rounded-lg bg-gray-200">
      <img 
        src={src || Cover} 
        alt="Album cover" 
        className="h-full w-full object-cover"
      />
    </div>
  );
}