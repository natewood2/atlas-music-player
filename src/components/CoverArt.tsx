import Cover from "/images/cover-art.png";

export default function CoverArt({ src }: { src?: string }) {
  console.log("CoverArt src:", src);
  return (
    <div className="w-full overflow-hidden rounded-lg bg-gray-200">
      <img 
        src={src || Cover} 
        alt="Album cover" 
        className="h-full w-full object-cover"
      />
    </div>
  );
}