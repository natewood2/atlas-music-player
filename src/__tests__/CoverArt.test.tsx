import { it, expect } from 'vitest';
import { render } from '@testing-library/react';
import CoverArt from '../components/CoverArt';

it('renders CoverArt', () => {
  const { container } = render(<CoverArt src="test-image.jpg" />);
  expect(container).toMatchSnapshot();
});