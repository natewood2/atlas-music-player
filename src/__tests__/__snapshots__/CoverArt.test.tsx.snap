// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders CoverArt 1`] = `
<div>
  <div
    class="w-full overflow-hidden rounded-lg bg-gray-200"
  >
    <img
      alt="Album cover"
      class="h-full w-full object-cover"
      src="test-image.jpg"
    />
  </div>
</div>
`;
